lib/tasks/locale.rake

```diff
@@ -220,6 +220,15 @@ namespace :locale do
           'ja'    => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/ja/ui-components.po',
           'pt_BR' => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/pt_BR/ui-components.po',
           'zh_CN' => 'https://raw.githubusercontent.com/ManageIQ/ui-components/master/locale/zh_CN/ui-components.po',
+        },
+
+        'react-ui-components' => {
+          'en'    => 'https://raw.githubusercontent.com/ManageIQ/react-ui-components/master/locale/en/ui-components.po',
```

Comment:
Actually, not OK... the pot file is called react-ui-components.pot, and I'm not seeing any renames in https://github.com/ManageIQ/react-ui-components/pull/125.

So... this should be `locales/en/react-ui-components` instead of just `locales/en/ui-components`, right?

Reply:
You're right, I missed that, thanks. Fixed now.